### ISSUE TITLE
Can't attach event listeners to the changeset object

### ISSUE THREAD

**Issue by ekawatani** [poteto/ember-changeset#172]
I can't seem to attach events to a changeset object. In the [demo](http://bit.ly/ember-changeset-demo),  added a js file for the `validated-field` component, and added the `afterValidation` listener to the changeset passed to the component.

```javascript
// app/components/validated-field.js
import Ember from 'ember';

export default Ember.Component.extend({
    init() {
        this._super(...arguments);

        let changeset = this.get('changeset');
        changeset.on('afterValidation', key => {
          console.log(`${key} has completed validating`);
        });
    }
});
```

Looks great, but I get `TypeError: changeset.on is not a function` in the console. Isn't this how I should do this?

I want to use the `afterValidation` event because I want to attach a class name to each invalid field's component root element. (Maybe there's a better way?)

**Comment by SuperCoolFrog** [comment]
@ekawatani looks like the latest release does not include that change. 
@poteto Would it take another PR to update the release?

**Comment by ekawatani** [comment]
Thanks, that'll unblock me for now.

**Comment by erichonkanen** [comment]
hello, I am getting the same issue, how can I use the `afterValidation` event?

```
  init() {
    this._super(...arguments);
    this.get('changeset').on('afterValidation', this, function() {
      debugger;
    });
  },
```

**Comment by khornberg** [comment]
Broken for me too

**Comment by khornberg** [comment]
Installing from git works.

A note for others. I had both `ember-changeset` and `ember-changeset-validations` installed under node 7.8.0. For some reason, `ember-changeset-validations` was using a version of `ember-changeset` local to itself. Reinstalled node modules and moved to node stable resolved the node issue.

**Comment by erichonkanen** [comment]
@khornberg will try that out, thanks

**Comment by erichonkanen** [comment]
@ekawatani following @khornberg advice I got it to work.. try rebuilding after wiping node_modules using latest...

**Comment by nucleartide** [comment]
Sounds like everything here is resolved. Please [create a new issue](https://github.com/poteto/ember-changeset/issues/new) if you run into more problems.
